OpenEdXMobile/src/main/java/org/edx/mobile/model/db/DownloadEntry.java

```diff
@@ -48,6 +48,12 @@
     public String lmsUrl;
     public TranscriptModel transcript;
 
+    /**
+     * TODO: Add a field in the {@link org.edx.mobile.module.db.DbStructure.Table#DOWNLOADS} database
+     * table to store the videoThumbnail along the video info.
+     */
```

Comment:
Add a particular case why do we need it, for example, add for deep linking we need to do it.

Reply:
Done.